### PR TITLE
[CHORE] Update minor and patch deps

### PR DESCRIPTION
**Pull request by a6software**
https://dsdmoj.atlassian.net/browse/SI-1193

https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by Stubbs — APPROVED** [review]

File: server/views/accessibility-statement.html [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
-<div class="govuk-breadcrumbs">
+<nav class="govuk-breadcrumbs" aria-label="Breadcrumb">
     <ol class="govuk-breadcrumbs__list">
         <li class="govuk-breadcrumbs__list-item">
             <a class="govuk-breadcrumbs__link" href="/">Home</a>
         </li>
         <li class="govuk-breadcrumbs__list-item" aria-current="page">Accessibility statement for the Digital Categorisation Service</li>
     </ol>
-</div>
+</nav>
```
Comment (a6software) [inline server/views/accessibility-statement.html:14]: as per changelog for 5.5.0 of gov uk front end (link in pr description)

File: server/views/partials/header.njk [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {%- from "tag/macro.njk" import govukTag -%}
 
-<header class="hmpps-header" role="banner" data-qa="cat-tool-fallback-header">
+<header class="hmpps-header" data-qa="cat-tool-fallback-header">
```
Comment (a6software) [inline server/views/partials/header.njk:3]: as per changelog for 5.5.0 of gov uk front end (link in pr description)

File: test/routes/liteCategories.test.js [hunk 3]
```diff
@@ -70,9 +70,13 @@ describe('assessment', () => {
         expect(res.text).toContain('Other category assessment</h1>')
         expect(res.text).toContain('<option value="RECP">Reception</option>')
         expect(res.text).toContain('<option value="SECUR">Security</option>')
-        expect(res.text).toContain(
-          `<input class="govuk-input govuk-!-width-full" id="nextReviewDate" name="nextReviewDate" type="text" value="${sixMonths}">`
+        expect(res.text).toMatch(
+          new RegExp(
+            `<input\\s+class="govuk-input govuk-!-width-full"\\s+id="nextReviewDate"\\s+name="nextReviewDate"\\s+type="text"\\s+value="${sixMonths}"\\s*>`,
+            's'
+          )
```
Comment (a6software) [inline test/routes/liteCategories.test.js:77]: test failed when the date forced the `input` tag to render on a separate line to the rest of the inner stuff

switched to regex to ignore whitespace

**Review by timothy-ness — APPROVED** [review]

**Comment by a6software** [comment]
thanks

**Pull request merged**